Gemfile

```diff
@@ -4,8 +4,9 @@ gemspec
 gem 'json', '~> 1.8.3'
 gem 'rest-client', '~> 1.8.0'
 gem 'activesupport', '~> 4.2.3'
+gem 'pmckee11-redis-lock', require: 'redis-lock'
```

Comment:
Will requiring it in the file it's used skip that issue? (That option does the require as part of bundler's powers IIRC, not sure it's desirable in a gem)